Office 365/o365/ingest/parser.yml

```diff
@@ -320,6 +318,21 @@ stages:
           office365.investigation.alert.correlation_key: "{{parse_data.ParsedData.CorrelationKey}}"
           office365.investigation.alert.category: "{{parse_data.ParsedData.Category}}"
           office365.investigation.alert.source_type: "{{parse_data.ParsedData.SourceAlertType}}"
+          event.start: "{{json_event.message.StartTimeUtc}}"
+          event.end: "{{json_event.message.EndTimeUtc}}"
           host.name: "{{parse_data.ParsedData.MachineName}}"
+          log.level: "{{parse_data.ParsedData.Severity}}"
+          rule.name: "{{parse_data.ParsedData.AlertDisplayName}}"
+
       - set:
           event.kind: '{% if parse_data.ParsedData.IsIncident %}{{"alert"}}{% else %}{{"event"}}{% endif %}'
+
+      - set:
+          event.url: '[{% for Property in parse_data.ParsedData.Entities %}{% for Url in Property.Urls %} "{{Url}}", {% endfor %}{% endfor %}]'
+          source.ip: '[{% for Property in  parse_data.ParsedData.Entities %} "{{Property.SenderIP}}", {% endfor %}]'
+          email.subjects: '[{% for Property in parse_data.ParsedData.Entities %} "{{Property.Subject}}", {% endfor %}]'
+          email.from.domain: '[{% for Property in parse_data.ParsedData.Entities %} "{{Property.P1SenderDomain}}", {% endfor %}]'
+          email.from.address: '[{% for Property in parse_data.ParsedData.Entities %} "{{Property.Sender}}", {% endfor %}]'
+          threat.enrichments: '[{% for Property in parse_data.ParsedData.Entities %} "{{Property.Threats}}", {% endfor %}]'
```

Comment:
[`threat.enrichments`](https://www.elastic.co/guide/en/ecs/8.6/ecs-threat.html#field-threat-enrichments) expects a list of [`indicators`](https://www.elastic.co/guide/en/ecs/8.6/ecs-threat.html#field-threat-enrichments-indicator).
I think the values are not suitable for this field. I suggest creating a custom field `office365.investigation.threats`.